From bc0118889db3c48fd5760fcd8d9633373a18d8fc Mon Sep 17 00:00:00 2001
From: Eyal Reizer <eyalr@ti.com>
Date: Wed, 30 Mar 2011 15:39:26 +0200
Subject: [PATCH 153/153] allow selecting WL12XX_PLATFROM_DATA independently of whether mac80211
 and wl12xx drivers are selected in the kernel.
 Needed for building with compat-wireless

Signed-off-by: Eyal Reizer <eyalr@ti.com>
---
 arch/arm/mach-omap2/Kconfig         |    1 +
 drivers/net/wireless/wl12xx/Kconfig |    1 -
 2 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-omap2/Kconfig b/arch/arm/mach-omap2/Kconfig
index c787aef..0f67445 100644
--- a/arch/arm/mach-omap2/Kconfig
+++ b/arch/arm/mach-omap2/Kconfig
@@ -37,6 +37,7 @@ config ARCH_OMAP3
	select ARM_L1_CACHE_SHIFT_6 if !ARCH_OMAP4
	select ARCH_HAS_OPP
	select PM_OPP if PM
+	select WL12XX_PLATFORM_DATA

 config ARCH_OMAP4
	bool "TI OMAP4"
diff --git a/drivers/net/wireless/wl12xx/Kconfig b/drivers/net/wireless/wl12xx/Kconfig
index b447559..441aaf6 100644
--- a/drivers/net/wireless/wl12xx/Kconfig
+++ b/drivers/net/wireless/wl12xx/Kconfig
@@ -42,5 +42,4 @@ config WL1271_SDIO

 config WL12XX_PLATFORM_DATA
	bool
-	depends on WL1271_SDIO != n
	default y
--
1.7.0.4
